addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -821,6 +830,22 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         // We need to reload the existing type list.
         this.rerender = true;
     },
+    /**
+     * Generates new ids for the cloned elements
+     * 
+     * @override
+     */
+    onClone() {
+        const clonedEl = this.$target[0];
+        const newId = this._generateUniqueID();
+        for (const el of clonedEl.querySelectorAll('[id] :not(div)')) {
```

Comment:
What you wanted here is probably
```suggestion
        for (const el of clonedEl.querySelectorAll('[id]:not(div)')) {
```
This is adding the ID to every element, not only the input
![image](https://user-images.githubusercontent.com/30048408/128509711-d0642c37-887f-4865-92d0-efe0ee8ccc83.png)

![image](https://user-images.githubusercontent.com/30048408/128509753-f27f040e-ec0e-422a-bd56-8556668c6357.png)

Reply:
By the way, why `:not(div)` is needed?
Is there a reason to duplicate a div id?

Reply:
Hi, if I remember correctly, I had put this because when we are in edit mode, the fields of types select options are replaced by divs which are themselves wrapped in a div which id = editable select.
![whyId](https://user-images.githubusercontent.com/78849981/128734498-82461476-54f2-4d57-ae12-3f0c66c8452a.png)

Reply:
Ok thank you, then it should at least be a comment to explain that in the code
IMHO, this was a mistake from the existing code to use an id and not the class to do that :thinking: